### PR TITLE
Swift 3.0

### PR DESCRIPTION
**Pull request by ikesyo**
This succeeds the @norio-nomura's work of #510.

### CODE REVIEW

File: .swift-version [hunk 1]
```diff
@@ -1 +1 @@
-DEVELOPMENT-SNAPSHOT-2016-02-25-a
+3.0-GM-CANDIDATE
```
Comment (kylef) [inline .swift-version]: This can be updated to `3.0` now.

**Comment by dannypier** [comment]
Is there a way to point my Pod at the nimble branch specified here? Not sure how that works or doesn't work differently than Swift Package Manager.

**Comment by dannypier** [comment]
I was able to find the answer to my question here: https://github.com/Quick/Nimble/issues/338

**Comment by yas375** [comment]
@ikesyo @norio-nomura thanks for putting it all together! 👍 

I've just tried to use this branch on a sample project with CocoaPods integration. 

```ruby
pod 'Quick', git: 'https://github.com/Quick/Quick.git', commit: '1503fb0'
```

When I launch Xcode it still wants to convert it to Swift 3 and it finds somethings:



I uncheck all the files it wants to modify and everything compiles and seem to work! Great job 👏 Can't wait to see it merged in!

**Comment by ikesyo** [comment]
Nimble is updated to v5.0.0 🎉

**Comment by jeffh** [comment]
We'll need to indicate the swift version to suppress the migrator from wanting to migrate Quick. Something like this:

```diff
--- a/Quick.xcodeproj/project.pbxproj
+++ b/Quick.xcodeproj/project.pbxproj
@@ -1858,6 +1858,7 @@
                                PRODUCT_NAME = "$(TARGET_NAME)";
                                SWIFT_OBJC_BRIDGING_HEADER = Sources/QuickTests/Helpers/QuickTestsBridgingHeader.h;
                                SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+                               SWIFT_VERSION = 3.0;
                        };
                        name = Debug;
                };
@@ -1878,6 +1879,7 @@
                                PRODUCT_NAME = "$(TARGET_NAME)";
                                SWIFT_OBJC_BRIDGING_HEADER = Sources/QuickTests/Helpers/QuickTestsBridgingHeader.h;
                                SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+                               SWIFT_VERSION = 3.0;
                        };
                        name = Release;
                };
@@ -2041,6 +2043,7 @@
                                PRODUCT_MODULE_NAME = Quick;
                                PRODUCT_NAME = Quick;
                                SKIP_INSTALL = YES;
+                               SWIFT_VERSION = 3.0;
                                VALID_ARCHS = x86_64;
                        };
                        name = Debug;
@@ -2067,6 +2070,7 @@
                                PRODUCT_NAME = Quick;
                                SKIP_INSTALL = YES;
                                SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+                               SWIFT_VERSION = 3.0;
                                VALID_ARCHS = x86_64;
                        };
                        name = Release;
@@ -2091,6 +2095,7 @@
                                PRODUCT_NAME = "$(TARGET_NAME)";
                                SWIFT_OBJC_BRIDGING_HEADER = Sources/QuickTests/Helpers/QuickTestsBridgingHeader.h;
                                SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+                               SWIFT_VERSION = 3.0;
                        };
                        name = Debug;
                };
@@ -2110,6 +2115,7 @@
                                PRODUCT_NAME = "$(TARGET_NAME)";
                                SWIFT_OBJC_BRIDGING_HEADER = Sources/QuickTests/Helpers/QuickTestsBridgingHeader.h;
                                SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+                               SWIFT_VERSION = 3.0;
                        };
                        name = Release;
                };
```

**Comment by ikesyo** [comment]
@jeffh That is set at the project level, not at each target.

**Comment by jeffh** [comment]
@ikesyo, I think that should be fine then. I ran it through the migrator to see what changed.

The current changes looks good to merge. I'll verify all configurations before cutting a release.

**Comment by doozMen** [comment]
👌🏻

**Comment by pcantrell** [comment]
Smoke tested with Siesta. 👍 

```
Test Suite 'All tests' passed at 2016-09-22 14:37:31.141.
	 Executed 331 tests, with 0 failures (0 unexpected) in 2.775 (3.127) seconds
```

**Pull request merged**